frontend/mr-admin-flate/src/components/avtaler/AvtaleSkjemaDetaljer.tsx

```diff
@@ -79,6 +79,12 @@ export function AvtaleSkjemaDetaljer({ tiltakstyper, ansatt, enheter, avtale }:
       label: `${enhet.navn} - ${enhet.organisasjonsnummer}`,
     }));
 
+    if (leverandorData) {
+      options.push({ label: leverandorData?.navn, value: leverandorData?.organisasjonsnummer });
+    } else {
+      options.push({ label: watchedLeverandor, value: watchedLeverandor });
+    }
```

Comment:
Jeg så ikke at leverandorData fantes når jeg la til den under tror jeg 🤔 For da er det vel ikke vits med den valgtLeverandor useState'en lenger? Jeg antar det blir to innslag i lista når det er sånn som dette nå?

Reply:
Fjerner staten siden klarer oss med leverandorData